### PR TITLE
add blogpost June meetup

### PR DESCRIPTION
**Pull request by liekelotte**

### CODE REVIEW

File: _posts/2021-07-07-JdlC-meetup-June.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+title: "What Open Science can learn from Free Software"
+excerpt_separator: <!--break-->
+date: 2021-07-07
+---
+Photo by <a href="https://unsplash.com/@rocinante_11?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Mick Haupt</a> on <a href="https://unsplash.com/s/photos/wall-garden?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
+
+Authors:  
+[Lieke de Boer](https://twitter.com/liekedb) / Netherlands eScience Center  
+
+[Javier de la Cueva](https://twitter.com/jdelacueva) does not like slideshows – &quot;_it limits the ways we communicate, like squeezing thoughts into 140 characters_&quot;, but for the NL-RSE meetup, he made one anyway. De la Cueva is a lawyer, philosopher, and Open Science advocate, with expertise in free software and intellectual property. During the meetup on Friday, 11 June 2021, he gave a passionate talk about what the Open Science movement can learn from the Free Software movement, blending the topics of software development and science in a way not unfamiliar to those identifying as RSEs.
```
Comment (c-martinez) [inline _posts/2021-07-07-JdlC-meetup-June.md:13]: I think he shared his slides, right? Do you want to link to them from the post?

Comment (c-martinez) [reply _posts/2021-07-07-JdlC-meetup-June.md:13]:
```suggestion
[Javier de la Cueva](https://twitter.com/jdelacueva) does not like slideshows – &quot;_it limits the ways we communicate, like squeezing thoughts into 140 characters_&quot;, but for the NL-RSE meetup, he made one anyway. De la Cueva is a lawyer, philosopher, and Open Science advocate, with expertise in free software and intellectual property. During the [meetup on Friday, 11 June 2021](/events/2021-06-11-meetup.html), he gave a passionate talk about what the Open Science movement can learn from the Free Software movement, blending the topics of software development and science in a way not unfamiliar to those identifying as RSEs.
```

File: _posts/2021-07-07-JdlC-meetup-June.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
...itched from Windows to Linux in 2000, after receiving a Blue Screen of Death on his Windows machine – &quot;_Why would you create a wall that doesn&#39;t allow me to understand something I paid for?_&quot;. In 2002, when Wikipedia was planning to commercialize, De la Cueva and others [forked the Spanish version of the page onto a server in the University of Seville, Spain](https://www.wired.co.uk/article/wikipedia-spanish-fork), proving to Wikipedia&#39;s owners that the entire project could be run on non-profit servers – a move some consider responsible for keeping Wikipedia free of commercial ads and paywalls to this day. Later, in 2006, he was, in his own words, &quot;_the first lawyer_ _to obtain a judicial ruling where the term _[_copyleft_](https://copyleft.org/)_ was included_&quot;. For the case in question, [Ladinamo](https://edri.org/our-work/edrigramnumber4-9spaincc/), De la Cueva defended the owners of a bar who were sued for failing to pay the compulsory licensing fee for playing music in their establishment. He won the case by successfully arguing that the music&#39;s Creative Commons licenses allowed for public performance, and that no additional fees had to be paid.
```
Comment (c-martinez) [inline _posts/2021-07-07-JdlC-meetup-June.md:15]: Not sure how you wanted your __copyleft__ to render, but I feel there might be one too many _ before the quote.

File: _posts/2021-07-07-JdlC-meetup-June.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+We&#39;d like to thank Javier de la Cueva for his talk on Open Science. This talk was part of the June NL-RSE meetup. NL-RSE meetups are monthly events, where those working on the intersection between research and software development (from PhD students to Scientific Programmers) at Dutch institutes come together to discuss relevant topics. Our next NL-RSE meetup is on September 16 and will cover reproducible research articles. Speakers include Fernando Pérez, creator of the IPython programming environment, Giulia Guizzardi, Innovation Officer at eLife, and Adina Wagner, author of the software package Datalad. [More information and sign-up here](https://nl-rse.org/events/2021-09-15-meetup.html).
```
Comment (c-martinez) [inline _posts/2021-07-07-JdlC-meetup-June.md:27]: Maybe link to the tweeter accounts and/or projects (Jupyter, eLife, Datalad, etc).

**Review by c-martinez — APPROVED** [review]
Left a few minor comments, but feel free to merge when you are ready.

File: _posts/2021-07-07-JdlC-meetup-June.md (hunk 3 above)
Comment (liekelotte) [reply _posts/2021-07-07-JdlC-meetup-June.md:27]: > Left a few minor comments, but feel free to merge when you are ready.

I'm going to merge now, and will put adding the links / slides on my (longish) to-do list!

**Pull request merged**